project.php

```diff
@@ -1595,7 +1599,7 @@ function do_post_files()
 }
 
 // -----------------------------------------------------------------------------
-function echo_upload_backup($projectid, $text, $stage)
+function echo_upload_backup(?string $projectid, string $text, string $stage): void
```

Comment:
`$projectid` should not be null and if it that's probably a bug.

Reply:
Makes sense, I don't remember why I allowed it but reverted this.